### PR TITLE
Update zh_CN.lang to translate new entries

### PR DESCRIPTION
**Pull request by RisingInIris2017**
With the BL updated to Release 3.3.8, there are several new entries of configuration options. This update will translate them into Chinese. Besides, some entries are replaced with better ones.

### CODE REVIEW

File: src/main/resources/assets/thebetweenlands/lang/zh_CN.lang [hunk 1]
```diff
@@ -1070,19 +1070,26 @@ command.blevent.failure.alreadydisabled=天气事件已经被禁用了。
...
 
+command.aspect.reset.usage=指令格式: /resetAspects
 command.aspect.reset.confirm=输入 '/resetAspects confirm'以确认。
 command.aspect.reset.success=重设要素完成。
 
+command.blreloadrecipes.usage=指令格式: /blReloadRecipes
 command.blreloadrecipes.success=重新载入自定义配方。
 command.blreloadrecipes.failed=载入自定义配方失败。
 
+command.set_decay.usage=指令格式: /setDecay <player> <decay> [saturation]
```
Comment (TheCyberBrick) [inline src/main/resources/assets/thebetweenlands/lang/zh_CN.lang]: You may want to replace \<player\> here with the same word you used for /aspectDiscovery. \<decay\> and [saturation] are also variables so feel free to translate those too.

**Comment by TheCyberBrick** [comment]
Thanks!

**Comment by TheCyberBrick** [comment]
By the way, I've found out that config tooltips can be translated too by appending ".tooltip" to the lang key.  For example adding
> `config.thebetweenlands.dimension_id.tooltip=This is now the tooltip for the dimension ID entry`

**Comment by RisingInIris2017** [comment]
Thanks for your advice! I will modify them as soon as possible :>

**Pull request merged**